### PR TITLE
cleanup with comprhensions, fstrings, deprecated classes, ...

### PR DESCRIPTION
**Pull request by wdpypere**

### CODE REVIEW

File: lib/vsc/administration/vo.py [hunk 1]
```diff
@@ -159,14 +158,15 @@ def sharing_group(self):
...
-                                   "Could not get autogroup %s details" % group_name)[1])
+                                   f"Could not get autogroup {group_name} details")[1])
 
         return self._sharing_group_cache
 
     @property
     def data_sharing(self):
         return self.vo_data_shared_quota is not None
 
+    @property
```
Comment (wdpypere) [inline lib/vsc/administration/vo.py:169]: @itkovian I don't know what happened here, but tests were failing on both py36 and 39. It seems this line was forgotten? But I don't understand how tests could have worked before without it.

(also a bunch of attributes were missing in test data)

Comment (wdpypere) [reply lib/vsc/administration/vo.py:169]: ah, I understand how it got missed. it only fails when you have a object with missing attributes (isactive was missing in some testdata, triggering the bug).

Comment (wdpypere) [reply lib/vsc/administration/vo.py:169]: it either the `@property` or the `error_vos[vo.vo_id] = vo.members` that needs to become a function call (`.members()`) on line 568

File: bin/create_tier2_ugent_home_data_directory_tree.py [hunk 2]
```diff
@@ -66,7 +66,7 @@ def main():
 
     storage_settings = VscStorage()
 
-    local_storage_conf = configparser.SafeConfigParser()
+    local_storage_conf = configparser.ConfigParser()
```
Comment (stdweird) [inline bin/create_tier2_ugent_home_data_directory_tree.py:69]: why is this needed?

File: lib/vsc/administration/vo.py [hunk 3]
```diff
@@ -108,7 +107,7 @@ def _institute_quota(self):
         if not self._institute_quota_cache:
             all_quota = [mkVscVoSizeQuota(q) for q in
                          whenHTTPErrorRaise(self.rest_client.vo[self.vo.vsc_id].quota.get,
-                                            "Could not get quotata from accountpage for VO %s" % self.vo.vsc_id)[1]]
+                                            f"Could not get quotata from accountpage for VO {self.vo.vsc_id}")[1]]
```
Comment (stdweird) [inline lib/vsc/administration/vo.py:110]: quota typo

File: bin/create_tier2_ugent_home_data_directory_tree.py (hunk 2 above)
Comment (wdpypere) [reply bin/create_tier2_ugent_home_data_directory_tree.py:69]: safeconfigparser was renamed to configparser in python3. the old "unsafe" configparser is gone, and safeconfigparser is deprecated.

File: lib/vsc/administration/vo.py (hunk 1 above)
Comment (wdpypere) [reply lib/vsc/administration/vo.py:169]: @stdweird  in `process_vos` on line 568, it has `error_vos[vo.vo_id] = vo.members`. it should be `error_vos[vo.vo_id] = vo.members()`.

**Comment by wdpypere** [comment]
@wpoely86 your replacing of rheia by thea in vsc-config broke vsc-administration. :smile:  You should have added it instead of replacing it I guess. This now incorporates that change.

**Comment by wpoely86** [comment]
> @wpoely86 your replacing of rheia by thea in vsc-config broke vsc-administration. 😄 You should have added it instead of replacing it I guess. This now incorporates that change.

No, it is being replaced. We just have not yet changed it everywhere. Thanks for doing it here :wink:

File: lib/vsc/administration/vo.py (hunk 1 above)
Comment (wdpypere) [reply lib/vsc/administration/vo.py:169]: hmm, the more I think about it, the I feel like the 2nd option is the better one.

Comment (wdpypere) [reply lib/vsc/administration/vo.py]: I switched it around and fixed the vo.members call
